src/setup.py

```diff
@@ -0,0 +1,24 @@
+"""The Setup File for our Mailroom App."""
+
+
+from setuptools import setup
+
+setup(
+    name="Mailroom",
+    description="An implementation of the Mailroom Module",
+    version=0.1,
+    author=["Conor Clary", "Regenal Grant"],
+    author_email=["sclary50@gmail.com", "regenal@mac.com"],
+    license="MIT",
+    package_dir={'': './'},
```

Comment:
This right here is a bit of a problem. The value in that key/value pair is the directory of code which the installer will try to install. This should be your `src` directory, not its parent.